int16 state
float64 sweep_speed
float64 x
float64 y
float64 yaw
float64 probe_angle
bool position_reached
